src/bucket/BucketList.cpp

```diff
@@ -24,71 +24,91 @@
 namespace stellar
 {
 
-BucketLevel::BucketLevel(uint32_t i)
+template <> BucketListDepth BucketListBase<LiveBucket>::kNumLevels = 11;
+
+// TODO: I made this number up, do some analysis and pick a better value or
+// make this a configurable network config.
```

Comment:
Just curious, can we actually change this value at runtime in-between ledgers?

Reply:
No. We will probably end up making this a config setting at which point I'll change it to a dynamic runtime value, but for now it's easier just to inherit the older BucketList's global const value for this refactor.